tyrian/js/src/main/scala/tyrian/package.scala

```diff
@@ -0,0 +1,9 @@
+package tyrian
+
+extension (l: Location.type)
```

Comment:
I'd encode this as a platform trait instead of an extension. The extension will only work if the user does `import tyrian.*`  and not if they explicitly import things.

The platform trait encoding looks like
```scala
object Location extends LocationPlatform

trait LocationPlatform: // JS
  def fromJsLocation(...): ...
  
trait LocationPlatform // JVM
  // empty

Reply:
Yes, that's a much better idea. Thanks!